data/json/recipes/weapon/explosive.json

```
@@ -14,6 +14,19 @@
       [ [ "cordage_short", 1, "LIST" ] ]
     ]
   },
+  {
+    "result": "c4",
+    "type": "recipe",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_EXPLOSIVE",
+    "skill_used": "fabrication",
+    "skills_required": [ "electronics", 3 ],
+    "difficulty": 4,
+    "time": "15 m",
+    "autolearn": true,
+    "components": [ [ [ "chem_rdx", 300 ] ], [ [ "e_scrap", 1 ] ], [ [ "cable", 10 ] ], [ [ "motor_oil", 50 ] ] ],
```

May be reasonable to add a bit more complex electronic components for this.
```suggestion
    "components": [
      [ [ "chem_rdx", 300 ] ],
      [ [ "delay_fuze", 1 ], [ "fuse", 1 ] ],
      [ [ "receiver", 1 ] ],
      [ [ "cable", 10 ] ],
      [ [ "motor_oil", 50 ] ]
    ],
```

I was tempted to even suggest allowing the `military_explosive` crafting requirement but I'm aware that C4 is specifically RDX so depends on if we want to be ultra-specific or more forgiving.